### PR TITLE
Boy_314's Halfnhalf layout update

### PR DESCRIPTION
**Pull request by Boy-314**
## Description

Added some commonly used macros to the Raise Layer of my half n half layout.

Also fixed some comment formatting with whitespaces

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [x] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c [hunk 1]
```diff
@@ -36,106 +43,135 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+			layer_off(_RAISE);
 			break;
 		case 3:
-			if (default_layer_state == DVORAK) {
-				default_layer_set(QWERTY);
-				layer_on(QWERTY);
-				layer_off(DVORAK);
+			if (default_layer_state == _DVORAK) {
+				default_layer_set(_QWERTY);
+				layer_on(_QWERTY);
```
Comment (drashna) [inline keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: Actually, you shouldn't need to turn on and off the layers here.  In fact, it's more harmful than helpful. 

https://docs.qmk.fm/#/keymap

Also: https://docs.qmk.fm/#/ref_functions?id=setting-the-persistent-default-layer

Comment (Boy-314) [reply keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: I tried the following: 
```		
case 3:
			if (default_layer_state == _DVORAK) {
				set_single_persistent_default_layer(_QWERTY);
			}
			else if (default_layer_state == _QWERTY) {
				set_single_persistent_default_layer(_DVORAK);
			}
			break;
```

But this only switches from Dvorak to Qwerty, and not back from Qwerty to Dvorak

Comment (drashna) [reply keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: Well, you need to use `biton32(default_layer_state)` there, not just the `default_layer_state`, and that should detect it properly.

Comment (drashna) [reply keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: Just to make sure, you've tested it, and it works how you want, correct?

Comment (Boy-314) [reply keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: Yea, it's working as intended now.

Comment (drashna) [reply keyboards/40percentclub/half_n_half/keymaps/Boy_314/keymap.c]: Awesome, glad to hear it!

**Pull request merged**